### ISSUE TITLE
create conda environment

### ISSUE THREAD

**Issue by nalbright** [dahak-metagenomics/dahak#103]
I am currently following the most recent installation workflow starting from clear slate to install everything from scratch: 

All is well until I get to _Create a Conda Environment_ When I execute:  _conda env create -f dahak-conda-environment.yml_ I receive: 
**SpecNotFound: Can't process without name**

I think it is looking for the .yml file, but cannot find one? Is there additional files that need installed to successfully create conda environment?  
Thanks!

**Comment by charlesreid1** [comment]
I added this command on suggestion from @stephenturner .  I'm just going to remove the `conda env create` step since it's not needed.

**Comment by nalbright** [comment]
Cool, thanks. I was able to finish the install directions successfully. The only thing that I had to stray away from the documentation was during the last install of the Open Science Framework CLI client using pip. When I executed this command I got the prompt:
_"mkl-random 1.0.1 and 1.0.2 requires cython, which is not installed"_
I fixed this just by doing a _pip install cython_ . Don't know if you want to add to documentation or not, but just a heads up. 
Thanks
